src/operator/sequence_last-inl.h

```diff
@@ -45,16 +45,46 @@ namespace op {
 namespace seq_last {
 enum SequenceLastOpInputs { kData, kSequenceLength };
 enum SequenceLastOpOutputs { kOut };
+enum SequenceLastOpResource { kTempSpace };
 }
 
 struct SequenceLastParam : public dmlc::Parameter<SequenceLastParam> {
   bool use_sequence_length;
+  uint32_t axis;
```

Comment:
Is it possible to support negative axis in the future? If so, it's better to define it as a signed integer.

Reply:
Yes. Fixed, thanks.